src/cms/models/pages/page_translation.py

```diff
@@ -144,6 +144,29 @@ def combined_last_updated(self):
             return mirrored_page_translation.last_updated
         return self.last_updated
 
+    @property
+    def tags(self):
+        """
+        This functions returns a list of translated tags which apply to this function.
+        Supported tags:
+        * Live content: if the page of this translation has live content
+        * Empty: if the page contains no text
+
+        :return: A list of tags which apply to this translation
+        :rtype: list [ str ]
+        """
+        tags = []
+
+        is_mirrored = self.page.mirrored_page is not None
+        if is_mirrored:
```

Comment:
```suggestion
        if self.page.mirrored_page:
```
Django automatically makes sure that the values of ForeignKey fields cannot evaluate to `False` if they are set to anything else than `None`. So it is safe to go without the additional variable here.